__tests__/app.test.js

```diff
@@ -72,6 +72,7 @@ describe('/api/articles/:article_id' , () => {
                 votes: 100,
                 article_img_url:
                   "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+                comment_count: 11
```

Comment:
As a rule of thumb, when we have wrote a test we shouldn't change it. We would expect to write another test to check for the comment count. This is also why we should write tests in a way that allow us to extend functionality later without compromising the tests.